<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseStartWindow</class>
 <widget class="QMainWindow" name="ChooseStartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monopoly</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="info_about_pl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>421</width>
      <height>119</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gloucester MT Extra Condensed</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>     Choose value of players:</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>419</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ok_btn_pl">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back_btn_pl">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>271</width>
      <height>178</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="radio_2_pl">
             <property name="font">
              <font>
               <family>Gloucester MT Extra Condensed</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>2 players</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_3_pl">
             <property name="font">
              <font>
               <family>Gloucester MT Extra Condensed</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>3 players</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="radio_4_pl">
             <property name="font">
              <font>
               <family>Gloucester MT Extra Condensed</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>4 players</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
